examples/ESP8266MeshHelloWorld/README.md

```diff
@@ -2,11 +2,35 @@
 
 This is a trivial node that can be used to test the mesh network.
 
-## Configuration
-a sample configuration is provided in `src/credentials.h.example`.
+![](http://i.imgur.com/w0uKBjU.png)
+
+If nodes can see the router, they communicate directly (Ex: 1 ->2). If not, they pass messages via the mesh network to other nodes. (Ex: 1 -> 2 -> 3)
+
+## QUICKSTART
+This describes one fast and easy way to get started, though there are other options.
+
+You will require:
+- A wirless router, with a dedicated secured (WPA2, AES) 2.4Ghz network, set to control channel 1 (2.412 Ghz), auto 20/40Mhz mode
```

Comment:
Channel 1 should no longer be necessary, since I've made the default be to use the staging tree